core/indexer/queries.go

```diff
@@ -17,11 +17,12 @@ func encode(obj object) (bytes.Buffer, error) {
 	return buff, nil
 }
 
-func txsByHashes(hashes []string) object {
+func documentsByIDs(hashes []string) object {
```

Comment:
Maybe the function name should have a verb in it

Reply:
renamed